examples/lustre/Dockerfile

```diff
@@ -3,18 +3,17 @@
 
 FROM centos7
 
-# install development tools
-RUN yum install -y epel-release
+# install development tools and dependencies
+RUN    yum install -y epel-release \
+    && yum -y install libyaml-devel \
+                      wget \
+                      gcc \
+    && yum clean all
 
-# more dependencies
-RUN yum -y install libyaml-devel \
-                   wget \
-                   gcc
-
-ENV LUSTRE_VERSION=2.12.3
-ENV LUSTRE_URL=https://downloads.whamcloud.com/public/lustre/lustre-${LUSTRE_VERSION}/el7/client/RPMS/x86_64/
+ARG LUSTRE_VERSION=2.12.3
+ARG LUSTRE_URL=https://downloads.whamcloud.com/public/lustre/lustre-${LUSTRE_VERSION}/el7/client/RPMS/x86_64/
 
 RUN    wget ${LUSTRE_URL}/lustre-client-${LUSTRE_VERSION}-1.el7.x86_64.rpm \
-    && wget ${LUSTRE_URL}/kmod-lustre-client-${LUSTRE_VERSION}-1.el7.x86_64.rpm
-
-RUN yum localinstall -y *.rpm
+    && wget ${LUSTRE_URL}/kmod-lustre-client-${LUSTRE_VERSION}-1.el7.x86_64.rpm \
+    && yum localinstall -y *.rpm \
```

Comment:
Why `yum localinstall` instead of `rpm` directly?

Reply:
It looks like `yum localinstall` will pull dependencies from repos whereas rpm will not. @Brofessional any thoughts?